test/E2ETests/Smoke.E2E.Tests/DataRowTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace MSTestAdapter.Smoke.E2ETests
+{
+    using Microsoft.MSTestV2.CLIAutomation;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    [TestClass]
+    public class DataRowTests : CLITestBase
+    {
+        private const string TestAssembly = "DataRowTestProject.dll";
+
+        [TestMethod]
+        public void ExecuteOnlyDerivedClassDataRowsWhenBothBaseAndDerviedClassHasDataRows()
+        {
+            this.InvokeVsTestForExecution(new string[] { TestAssembly }, testCaseFilter: "FullyQualifiedName~DerivedClass");
+
+            this.ValidatePassedTestsContain(
```

Comment:
Shouldn't this validation change after using the filter ?

Reply:
By mistake added filter in first test.